deploy/template.yaml

```
@@ -20,7 +17,7 @@ Parameters:
       - "staging"
       - "integration"
       - "production"
-    ConstraintDescription: must be dev, build, staging, integration or production
+    ConstraintDescription: Please ensure that your Environment variable is set to either dev, build, staging, integration or production.
```

```suggestion
    ConstraintDescription: Ensure environment variable is set to one of dev, build, staging, integration or production.
```

Prefer clear, concise instructions to avoid ambiguity